android/capacitor/src/main/java/com/getcapacitor/plugin/CapacitorHttp.java

```diff
@@ -31,8 +31,7 @@ private void http(final PluginCall call, final String httpMethod) {
             @Override
             public void run() {
                 try {
-                    HttpRequestHandler.bridge = bridge;
-                    JSObject response = HttpRequestHandler.request(call, httpMethod);
+                    JSObject response = HttpRequestHandler.request(call, httpMethod, getBridge());
```

Comment:
Replace storing a static reference to the Bridge with passing an instance of the Bridge in to the `request` call.